decidim-core/lib/decidim/exporters.rb

```diff
@@ -9,6 +9,11 @@ module Exporters
     autoload :PDF, "decidim/exporters/pdf"
     autoload :ExportData, "decidim/exporters/export_data"
 
+    # Lock the export formats to one of the available exporters
+    EXPORT_FORMATS = [:JSON, :CSV, :Excel, :PDF, :FormPDF].freeze
```

Comment:
I would extract this to a config_accessor. I am just thinking that we do not support "xml", "yaml" or any other formats.

Reply:
I made a new issue about this at #11629 to track it. I was also thinking that regarding the `FormPDF` export format which is provided by the `decidim-forms` gem and not the `decidim-core` gem as the other exporters. But there is already a condition to check that the module exists before returning it, so the current code also works if the `decidim-forms` gem is not available for one reason or another.

In this PR I won't change the current implementation because it adds more complexity and work to fixing this problem. I think the `config_accessor` approach is not sufficient if there are multiple modules registering their export format. Instead, I would imagine an export format registry which could be utilized by the 3rd party modules.